Server/network/packets/PlayerAuthInputPacket.hpp

```diff
@@ -0,0 +1,173 @@
+#ifndef BEDROCKPOWDER_PLAYERAUTHINPUTPACKET_HPP
+#define BEDROCKPOWDER_PLAYERAUTHINPUTACKET_HPP
+
+
+#include "Server/network/PacketType.hpp"
+#include "Server/network/MCPEPacket.hpp"
+
+#include "Server/utils/StdEnv.h"
+#include "Server/network/ProtocolInfo.h"
+
+struct Vec2
```

Comment:
We already have Vector2d in which everything is implemented.  It is not necessary to create a new structure.